### ISSUE TITLE
Error when compiling

### ISSUE THREAD

**Issue by myzinsky** [TUK-EIT/exam-eit-latex#5]
```
! LaTeX Error: Command \lhead already defined.
               Or name \end... illegal, see p.192 of the manual.
```

**Comment by cdeschryver** [comment]
Don't have this issue with 2f67508 when compiling with my IDE. Must be a problem with the Makefile. Will be fixed in the future ;)

**Comment by myzinsky** [comment]
The problem is this:
http://tex.stackexchange.com/questions/141296/conflict-between-exam-class-and-fancyhdr-package

and it should also happen in your IDE ...

**Comment by cdeschryver** [comment]
fixed in 9226198ddc28e6c5b9e634ca8d4ffcff3d3d23a4 by replacing scrpage2 with fancyhdr

**Comment by cdeschryver** [comment]
mmh. Still throws that error when trying to compile with the Makefile. IDE compiling works without issues. Anyone can check this on his/her system? Thanks!

**Comment by myzinsky** [comment]
I think the main clash is that fancyhdr and the exam package provide the same function. The solution would be to remove fancyhdr completely and use the provided function by the exam package.

**Comment by cdeschryver** [comment]
Agree. However, that would not allow a 1-to-1 use of the style-eit-latex submodule, since there we need fancyhdr. I'm still wondering why it compiles in my IDE, but not with the makefile...

**Comment by myzinsky** [comment]
The issue is fixed in #7
